documentation/modules/snip-migrating-luns.adoc

```diff
@@ -0,0 +1,10 @@
+:_content-type: SNIPPET
+
+[NOTE]
+====
+{project-short} supports direct migration of logical units numbers (LUNs) from {rhv-full} ({rhv-short}) source providers as follows:
+
+* LUNs are detached from the source provider and then attached to the target providers.
+
```

Comment:
I'm not sure if that's the right place for it but we can add something like:
LUNs are not removed from the source provider during the migration but remain detached in case fallback to the source provider is requied. However, before re-attaching the LUNS to virtual machines in the source provider please note that you need to make sure the LUNs are not attached to VMs on the target environments at the same time as this could lead to date corruption.